ports/stm32/boards/NUCLEO_WB55/rfcore_firmware.py

```diff
@@ -180,6 +180,11 @@
 _ADDR_FUS = 0x080EC000
 _ADDR_WS_BLE_HCI = 0x080DC000
 
+# When installing the WS it can take a long time to return to the first
+# GET_STATE HCI command.
+# e.g. Installing stm32wb5x_BLE_Stack_full_fw.bin takes 3600ms to respond.
+_INSTALLING_WS_GET_STATE_TIMEOUT = 6000
```

Comment:
const(6000)?

Reply:
Done (and added in a separate value for FUS)